composer.json

```diff
@@ -14,7 +14,7 @@
         "ext-json": "*",
         "fig/http-message-util": "^1.1.5",
         "psr/http-factory": "^1.0.2",
-        "psr/http-message": "^1.1.0",
+        "psr/http-message": "^1.1 || ^2.0",
```

Comment:
👋🏽 wondering if this is a WIP for the 3.3.x version, but it looks like `Lcobucci\ContentNegotiation\UnformattedResponse` is not compatible with the contracts of `Psr\Http\Message\ResponseInterface` psr/http-message:2.0.

Reply:
You're right!
Our pipeline didn't detect that due to dependency constraints 😥

https://packagist.org/packages/middlewares/negotiation needs a release to be compatible with v4 of https://packagist.org/packages/middlewares/utils. 

@oscarotero is that on your plans?